custom-tests.json

```diff
@@ -543,6 +543,279 @@
+      "__base": "var instance = document.createElementNS('http://www.w3.org/2000/svg', 'componenttransferfunction'); if (instance.constructor.name === 'SVGElement') {return false;}"
+    },
+    "SVGCursorElement": {
+      "__base": "var instance = document.createElementNS('http://www.w3.org/2000/svg', 'cursor'); if (instance.constructor.name === 'SVGElement') {return false;}"
+    },
+    "SVGDefsElement": {
+      "__base": "var instance = document.createElementNS('http://www.w3.org/2000/svg', 'defs'); if (instance.constructor.name === 'SVGElement') {return false;}"
+    },
+    "SVGDescElement": {
+      "__base": "var instance = document.createElementNS('http://www.w3.org/2000/svg', 'desc'); if (instance.constructor.name === 'SVGElement') {return false;}"
+    },
+    "SVGEllipseElement": {
+      "__base": "var instance = document.createElementNS('http://www.w3.org/2000/svg', 'ellipse'); if (instance.constructor.name === 'SVGElement') {return false;}"
+    },
+    "SVGFEBlendElement": {
+      "__base": "var instance = document.createElementNS('http://www.w3.org/2000/svg', 'feblend'); if (instance.constructor.name === 'SVGElement') {return false;}"
+    },
+    "SVGFEColorMatrixElement": {
+      "__base": "var instance = document.createElementNS('http://www.w3.org/2000/svg', 'fecolormatrix'); if (instance.constructor.name === 'SVGElement') {return false;}"
+    },
+    "SVGFEComponentTransferElement": {
+      "__base": "var instance = document.createElementNS('http://www.w3.org/2000/svg', 'fecomponenttransfer'); if (instance.constructor.name === 'SVGElement') {return false;}"
+    },
+    "SVGFECompositeElement": {
+      "__base": "var instance = document.createElementNS('http://www.w3.org/2000/svg', 'fecomposite'); if (instance.constructor.name === 'SVGElement') {return false;}"
+    },
+    "SVGFEConvolveMatrixElement": {
+      "__base": "var instance = document.createElementNS('http://www.w3.org/2000/svg', 'feconvolvematrix'); if (instance.constructor.name === 'SVGElement') {return false;}"
+    },
+    "SVGFEDiffuseLightingElement": {
+      "__base": "var instance = document.createElementNS('http://www.w3.org/2000/svg', 'fediffuselighting'); if (instance.constructor.name === 'SVGElement') {return false;}"
+    },
+    "SVGFEDisplacementMapElement": {
+      "__base": "var instance = document.createElementNS('http://www.w3.org/2000/svg', 'fedisplacementmap'); if (instance.constructor.name === 'SVGElement') {return false;}"
+    },
+    "SVGFEDistantLightElement": {
+      "__base": "var instance = document.createElementNS('http://www.w3.org/2000/svg', 'fedistantlight'); if (instance.constructor.name === 'SVGElement') {return false;}"
+    },
+    "SVGFEDropShadowElement": {
+      "__base": "var instance = document.createElementNS('http://www.w3.org/2000/svg', 'fedropshadow'); if (instance.constructor.name === 'SVGElement') {return false;}"
+    },
+    "SVGElement": {
+      "__base": "<%api.SVGSVGElement:instance%>"
```

Comment:
Isn't it possible to get a direct instance of `SVGElement`? If there is no `SVGUnknownElement`, then unrecognized elements in the parser probably become plain `SVGElement`s.